package.json

```diff
@@ -63,6 +63,7 @@
     "sinon": "^2.2.0",
     "sw-testing-helpers": "^1.0.2",
     "tmp": "0.0.31",
-    "uppercamelcase": "^3.0.0"
+    "uppercamelcase": "^3.0.0",
+    "webpack": "^2.6.0"
```

Comment:
I *think* that the Lerna best practice is to include this as a `devDependency` in `packages/workbox-webpack-plugin/package.json` instead of in the top-level `package.json`.